dir.props

```diff
@@ -92,7 +94,9 @@
     <OSPlatformConfig>$(BinDirOSGroup).$(BinDirPlatform).$(BinDirConfiguration)</OSPlatformConfig>
 
     <BaseOutputPath Condition="'$(BaseOutputPath)'==''">$(ProductBinDir)</BaseOutputPath>
-    <OutputPath Condition="'$(OutputPath)'==''">$(BaseOutputPath)$(OSPlatformConfig)/$(MSBuildProjectName)/</OutputPath>
+    <PackageOutputPath Condition="'$(PackageOutputPath)'==''">$(BaseOutputPath)pkg/$(OSPlatformConfig)/$(MSBuildProjectName)/</PackageOutputPath>
```

Comment:
Shouldn't you set PackageOutputPath relative to PackageOutputRoot that you're defining above?